### ISSUE TITLE
Spike Accessibility

### ISSUE THREAD

**Issue by vmarsterson** [vmarsterson/QuizApp#20]
## Description
**Given** that I am a developer

**When** I build my app

**I want to** have a basic understanding of accessibility

**So I can** incorporate inclusive design features into my application so that it can be used by as many users as possible 

## Time
Estimate: 2 days
Remaining: 2 days

## Acceptance Criteria
- [ ] Connect phone to Android Studio so I can get talkback working on a real device
- [ ] Download Accessibility Suit
- [ ] Test out a few apps and make notes on what works and what doesn't work
- [ ] Make a wiki page on accessibility explaining how to use testing suites and common practices / misconceptions
- [ ] Try to recreate Sky App bug and estimate remaining time
